src/App.jsx

```diff
@@ -1,16 +1,52 @@
 import logo from "/logo.png";
 import "./App.css";
+import { useState } from "react";
+import axios from "axios";
 
 function App() {
+  const [cityInput, setCityInput] = useState("");
+  const [cityTemperature, setCityTemperature] = useState("");
+
+  const handleSubmit = (event) => {
```

Comment:
Try to extract your key oout of the request. Either have a variable or use a .env file